api/lib/application/authentication/oidc/oidc-controller.js

```diff
@@ -4,8 +4,13 @@ import * as oidcSerializer from '../../../infrastructure/serializers/jsonapi/oid
 import { usecases } from '../../../domain/usecases/index.js';
 import { UnauthorizedError } from '../../http-errors.js';
 
+const getAllIdentityProvidersForAdmin = async function (request, h) {
+  const identityProviders = usecases.getAllIdentityProviders();
+  return h.response(oidcProviderSerializer.serialize(identityProviders)).code(200);
+};
+
 const getIdentityProviders = async function (request, h) {
-  const identityProviders = usecases.getIdentityProviders();
+  const identityProviders = usecases.getReadyIdentityProviders();
```

Comment:
Je serais plus pour nommer comme ceci

```suggestion
  const identityProviders = usecases.getEnabledIdentityProviders();
```

Reply:
On a hésite mais on a choisi `ready` parce que ça retourne les idp pour lesquels la propriété `isReady` est `true`, ce qui ne dépend pas du fait que l'idp soit "enabled", mais aussi que sa configuration soit valide.

Reply:
@igorissen c'est ce que j'avais proposé à Clément dans un premier mais il m'a expliqué que ce n'était pas suffisant de simplement indiqué que les idp sont enabled, pour les raisons citées au dessus